Source/WebKit/Shared/RTCPacketOptions.h

```diff
@@ -28,25 +28,34 @@
 #if USE(LIBWEBRTC)
 
 #include <WebCore/LibWebRTCMacros.h>
-#include <optional>
 #include <wtf/Forward.h>
+#include <wtf/Vector.h>
 
 ALLOW_COMMA_BEGIN
 
 #include <webrtc/rtc_base/async_packet_socket.h>
 
 ALLOW_COMMA_END
 
-namespace IPC {
-class Decoder;
-class Encoder;
-}
-
 namespace WebKit {
 
 struct RTCPacketOptions {
-    void encode(IPC::Encoder&) const;
-    static std::optional<RTCPacketOptions> decode(IPC::Decoder&);
+    struct SerializableData {
+        int32_t dscp;
+        int32_t packetId;
+        int rtpSendtimeExtensionId;
+        int64_t srtpAuthTagLength;
+        std::span<const uint8_t> srtpAuthKey;
+        int64_t srtpPacketIndex;
+    };
+
+    explicit RTCPacketOptions(const rtc::PacketOptions& options)
```

Comment:
Can we use a move constructor?

Reply:
Concurrently the call site is being given a `const rtc::PacketOptions&` so not easily. Maybe if I modify the call-site's call site. I'd have to look into it. At this this is not worse than before.

Reply:
The call site is:
```
int LibWebRTCSocket::SendTo(const void *value, size_t size, const rtc::SocketAddress& address, const rtc::PacketOptions& options);
```

It takes options as `const rtc::PacketOptions&`. The interface is defined by AsyncPacketSocket in the libWebRTC library so it is not something I can change. I can't pass an r-value here.